src/engine/Settings.cs

```diff
@@ -103,70 +151,177 @@ private Settings()
+        set => settingsData.MaxAutoSaves = value;
+    }
+
+    /// <summary>
+    ///   Number of quick saves to keep
+    /// </summary>
+    public int MaxQuickSaves
+    {
+        get => settingsData.MaxQuickSaves;
+        set => settingsData.MaxQuickSaves = value;
+    }
+
+    /// <summary>
+    ///   When true cheats are enabled
+    /// </summary>
+    public bool CheatsEnabled
+    {
+        get => settingsData.CheatsEnabled;
+        set => settingsData.CheatsEnabled = value;
+    }
+
+    /// <summary>
+    ///   If true an auto-evo run is started during gameplay,
+    ///   taking up one of the background threads.
     /// </summary>
-    public int ColourblindSetting { get; set; } = 0;
+    public bool RunAutoEvoDuringGamePlay
+    {
+        get => settingsData.RunAutoEvoDuringGamePlay;
+        set => settingsData.RunAutoEvoDuringGamePlay = value;
+    }
 
     public int CloudSimulationWidth => Constants.CLOUD_X_EXTENT / CloudResolution;
 
     public int CloudSimulationHeight => Constants.CLOUD_Y_EXTENT / CloudResolution;
 
     /// <summary>
-    ///   The amount of Chromatic Aberration to apply to the screen
+    ///   Loads and applies settings from the saved setting configuration file.
     /// </summary>
-    public float ChromaticAmount { get; set; } = 20f;
+    /// <returns>True on success, false if the file is invalid or couldn't be opened for reading.</returns>
+    public bool Load()
+    {
+        SettingsData loadedData;
+
+        var file = new File();
+
+        var error = file.Open(Constants.CONFIGURATION_FILE, File.ModeFlags.Read);
+
+        if (error != Error.Ok)
+        {
+            GD.Print("Settings configuration file is missing or unreadable.");
+            return false;
+        }
+
+        var text = file.GetAsText();
+
+        file.Close();
+
+        loadedData = JsonConvert.DeserializeObject<SettingsData>(text);
+
+        // Sets local data to the loaded data and automatically applies it.
+        Data = loadedData;
+        GD.Print("Loaded and applied settings from configuration file.");
+        return true;
+    }
 
     /// <summary>
-    ///   Enable or Disable Chromatic Aberration for screen
+    ///   Loads and applies default configuration settings.
     /// </summary>
-    public bool ChromaticEnabled { get; set; } = true;
+    public void LoadDefaults()
+    {
+        settingsData.ResetToDefaults();
+        ApplyAll();
+        GD.Print("Loaded and applied default settings.");
+    }
 
     /// <summary>
-    ///   Saves the current settings by writing them to the settings file
+    ///   Saves the current settings by writing them to the settings configuration file.
     /// </summary>
     /// <returns>True on success, false if the file can't be written.</returns>
     public bool Save()
     {
-        using (var file = new File())
-        {
-            var error = file.Open(Constants.CONFIGURATION_FILE, File.ModeFlags.Write);
+        var file = new File();
```

Comment:
File is a disposable type.

Reply:
Fair, I'll change that to using instead.